config/pipe_config.py

```diff
@@ -0,0 +1,3 @@
+# File paths for IPC pipes
```

Comment:
I was just thinking. Maybe we have a PIPE_PATH variable for where we want to store the pipes like "/FiberPTS/pipes" and a list of pipe file names assigned to PIPE_NAMES such as `['tapDataPipe', 'nfcDataPipe']`.

Reply:
Also maybe change the pipe names since they do specific things like `'tapDataPipe'` could be renamed to `'touchSensorToScreenPipe'` since the pipe describes an inter-process communication bridge between two specific processes. I know it doesn't look pretty, but it feels more functionally correct.

Reply:
I would not assume the `'/opt'` absolute path, just because 1. we don't need to and 2. it makes choosing a different directory for whatever reason significantly easier

Reply:
The reason is that pipes should only be a communication with a single process, so, for instance, if the screen is reading from `'tap_data_pipe'`, then no other process should read from that pipe ever to avoid concurrency issues since every read removes the data from the pipe (like dequeuing), so information may be lost to one of the concurrently reading processes.

Reply:
> I would not assume the `'/opt'` absolute path, just because 1. we don't need to and 2. it makes choosing a different directory for whatever reason a bit harder

Can you elaborate more on this? Not sure I understand.

Reply:
All that is needed for the pipe path is where in `'/FiberPTS'` the pipes will reside since we agreed that during setup, the pipes are just created within the FiberPTS application folder. No need to include where FiberPTS is, since that is just unnecessary information. If needed, can't you just use a relative pathing like `'../FiberPTS/pipes'`?